jocd/src/br/org/certi/jocd/dapaccess/Transfer.java

```diff
@@ -25,12 +25,29 @@
 
   private Exception error;
   private int sizeBytes = 0;
-  private int[] result;
+  private byte[] result;
```

Comment:
This change isn't right, the result of a transfer is an array of 4-byte words, so the correct would be to keep it as an int array.

Reply:
Fixed!